Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -93,7 +96,13 @@ private void OnToggleLawsScreen(EntityUid uid, SiliconLawBoundComponent componen
 
     private void OnBoundUIOpened(EntityUid uid, SiliconLawBoundComponent component, BoundUIOpenedEvent args)
     {
-        var state = new SiliconLawBuiState(GetLaws(uid));
+        _entityManager.TryGetComponent<IntrinsicRadioTransmitterComponent>(uid, out var intrinsicRadio);
+        _entityManager.TryGetComponent<SpeechComponent>(uid, out var speech);
+
+        bool canSpeak = speech?.Enabled == true;
+        HashSet<string> radioChannels = intrinsicRadio != null ? intrinsicRadio.Channels : new HashSet<string>();
+
+        var state = new SiliconLawBuiState(GetLaws(uid), canSpeak, radioChannels);
```

Comment:
Speech component is predicted on the client so CanVerbalize is unnecessary to network.